webviz_subsurface/plugins/_reservoir_simulation_timeseries.py

```diff
@@ -129,13 +120,23 @@ def __init__(
         ]
 
         self.ensembles = list(self.smry["ENSEMBLE"].unique())
-        self.plotly_theme = app.webviz_settings["theme"].plotly_theme
+        self.theme = app.webviz_settings["theme"]
         self.plot_options = options if options else {}
         self.plot_options["date"] = (
             str(self.plot_options.get("date"))
             if self.plot_options.get("date")
             else None
         )
+        line_shape_fallback = line_shape_fallback.lower()
+        if line_shape_fallback in ("backfilled", "backfill"):
+            self.line_shape_fallback = "vh"
+        elif line_shape_fallback in ["hv", "vh", "hvh", "vhv", "spline", "linear"]:
+            self.line_shape_fallback = line_shape_fallback
+        else:
+            self.line_shape_fallback = "linear"
+            warnings.warn(
+                f"{line_shape_fallback} is not a valid line_shape_fallback option, will use linear."
+            )
```

Comment:
Similar code blocks in multiple plugins. Move to common function?

Reply:
Could do that, not sure where it would naturally belong though? Doesn't quite fit neither `datainput` or `abbreviations` as I think of those terms.

Reply:
Not sure either. A new folder `_utils` containing small utility functions used by plugins not covered by the two existing subfolders?

Reply:
Think it is the cleanest option at least.